apps/docs/components/ui/mobileMenu/mobileMenu.css

```diff
@@ -0,0 +1,92 @@
+.hd-mobile-menu {
+    background: var(--hd-color-neutral-surface);
```

Comment:
I know we have all been putting the css variables directly where they are used, but I was thinking, weren't we supposed to have component level css variables that we set at the top in :root? kind of like we did in Igloo. Maybe I'm mistaken, not sure.

Reply:
Sure for the Hopper components. I am not sure we should go that far for the documentation website, as least not yet as the aim was to release something.